### PR TITLE
add sagemaker mxnet

### PR DESCRIPTION
**Pull request by Geoyi**
add Sagemaker MXNet image classification example.

### CODE REVIEW

File: examples/walkthrough-classification-mxnet-sagemaker.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- Log in your account, and go to the [sagemaker home page](https://console.aws.amazon.com/sagemaker/)
+- Create a Notebook Instance! ![screenshot 2017-12-20 17 20 42](https://user-images.githubusercontent.com/14057932/34264652-912868ea-e641-11e7-9877-60ede67eb421.png) Click on `Create notebook Instance`. You will have three instance options, `ml.t2.medium`, `ml.m4.xlarge` and `ml.p2.xlarge`, to choose from. We recommend you to use the p2 machine (a gpu machine) to train this image classification.
+
+Once you have your p2 instance notebook set up, you are ready to train a classifier. Specifically, you are going to learn how to plug your own script into Amazon SageMaker MXNet Estimator and train the classifier we prepared for detecting buildings in images.
+
+
+# Train the model with MXNet on AWS SageMaker
+Training a LeNet building classifier using MXNet Estimator:
+- Prepare your own training script, and you could use our `mx_lenet_sagemaker.py` here, just slightly modify it; You could see and follow from the [Jupyter Notebook we prepared](https://github.com/developmentseed/ml-data-generation/blob/mx_net/examples/nets/SageMaker_mx-lenet.ipynb).
```
Comment (drewbo) [inline examples/walkthrough-classification-mxnet-sagemaker.md]: @Geoyi can you link to the script and also update the links to this repo?

**Review by drewbo — CHANGES_REQUESTED** [review]
@Geoyi Just a small note on the links; then @wronk or I will try out the Sagemaker instructions 👍

File: examples/walkthrough-classification-mxnet-sagemaker.md (hunk 1 above)
Comment (Geoyi) [reply examples/walkthrough-classification-mxnet-sagemaker.md]: aww ... sure thing

**Comment by Geoyi** [comment]
@drewbo and @wronk, the link has updated and I've added the complete Jupyter Notebook. The result is not ideal, I think it's caused by:
1) smaller training dataset (560 tiles in total), and the quality of mapbox imagery at zoom 15 is varied;
2) Lenet is not an ideal algorithm for this type of complex images training task. 

Anyway, the case is up now for your test.

**Review by drewbo — APPROVED** [review]

**Pull request merged**